src/components/CreateMessageSelect.tsx

```diff
@@ -142,7 +142,7 @@ export const CreateMessageSelect = async () => {
             ))}
           </ul>
           <div class={checkLinkStyle}>
-            <a href="./check">決定</a>
+            <a id="message_save" onclick="messageSave()">決定</a>
```

Comment:
```suggestion
            <a id="message_save" onclick="_messageSave()">決定</a>
```
`script.js`と関数名がずれてそう

Reply:
修正しました